src/pages/Home.tsx

```diff
@@ -68,16 +94,21 @@ function Home() {
     setFile(acceptedFiles[0]);
     const fr = new FileReader();
     fr.readAsText(acceptedFiles[0]);
-    fr.onload = () => console.log(fr.result);
+    //    fr.onload = () => console.log(fr.result);
   }, []);
 
   const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
 
-  const { submit } = useSubmitTransaction({
+  const { submit: initialize } = useSubmitTransaction({
     functionName: 'initializeSarcophagus',
     contractInterface: EmbalmerFacet__factory.abi,
   });
 
+  const { submit: finalize } = useSubmitTransaction({
+    functionName: 'finalizeSarcophagus',
+    contractInterface: EmbalmerFacet__factory.abi,
+  });
```

Comment:
Recommend (eventually) all these contract interaction functions get moved to their own file.

Reply:
These will probably be added as dispatch actions in my current task.

Reply:
yes, all those will be in their own hooks soon  ....